entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -o errexit
+trap 'call_exit $LINE_NO' EXIT
+
+call_exit()
+{
+echo "at call_exit.."
+echo  "exit code:" $?
```

Comment:
I think that return value should be provided through `trap`.
`trap 'call_exit $? $LINE_NO' EXIT`.